src/Utils/StyleConfig.ts

```diff
@@ -59,11 +54,11 @@ export const ScreenSize = {
 };
 
 export const Device = {
-  mobileS: `(max-width: ${ScreenSize.mobileS}px)`,
-  mobileM: `(max-width: ${ScreenSize.mobileM}px)`,
-  mobileL: `(max-width: ${ScreenSize.mobileL}px)`,
-  tablet: `(max-width: ${ScreenSize.tablet}px)`,
-  desktopS: `(max-width: ${ScreenSize.desktopS}px)`,
-  desktopM: `(max-width: ${ScreenSize.desktopM}px)`,
-  desktopL: `(max-width: ${ScreenSize.desktopL}px)`,
+  mobileS: `(max-width: ${ScreenSize.mobileM - 1}px)`,
```

Comment:
Device is being used at some places in dst? These changes can affect those too.

Reply:
Right. I checked with `yalc` for DST too. Looks good to me, couldn't find any problems. There might be some slight differences but nothing is breaking, so let's merge this PR and deal with the problems as we go.

Reply:
Okay!

Reply:
Haha I don't think we can conclude that nothing breaks by just looking around the codebase. Since we swapped `max` with `min`, it should follow that all instances of `Device` used on our codebases would have to be refactored. https://glints.atlassian.net/browse/GM-780

Reply:
Oh also could someone help check the rest of the cases where `Device` is used? I suspect there may be other affected components.

Reply:
I'll talk to the designers about the change in the definition.

Reply:
another bug here, https://glints.atlassian.net/browse/GM-789
I am going to update all the Device references on DST

Reply:
@westwood846 , I think we should let the QA team check once after changes for components that have been used a lot in the projects. 🤔